src/test/java/cloud/fogbow/ras/core/plugins/interoperability/openstack/network/v2/OpenStackNetworkPluginTest.java

```diff
@@ -50,22 +54,17 @@
     private static final String FAKE_USER_ID = "fake-user-id";
     private static final String FAKE_NAME = "fake-name";
     private static final String FAKE_PROJECT_ID = "fake-project-id";
-
-    private static final String SUFFIX_ENDPOINT_DELETE_NETWORK = OpenStackNetworkPlugin.SUFFIX_ENDPOINT_NETWORK +
-            File.separator + NETWORK_ID;
-
-    private static final String SUFFIX_ENDPOINT_DELETE_SECURITY_GROUP = OpenStackNetworkPlugin.SUFFIX_ENDPOINT_SECURITY_GROUP +
-            File.separator + SECURITY_GROUP_ID;
-    private static final String SUFFIX_ENDPOINT_GET_SECURITY_GROUP = OpenStackNetworkPlugin.SUFFIX_ENDPOINT_SECURITY_GROUP + "?" +
-            OpenStackNetworkPlugin.QUERY_NAME + "=" + SystemConstants.PN_SECURITY_GROUP_PREFIX + NETWORK_ID;
+    private static final String FAKE_TENANT_ID = "tenant-id";
+    private static final String FAKE_SUBNET_ID = "subnet-id";
 
     private OpenStackNetworkPlugin openStackNetworkPlugin;
-    private OpenStackV3User openStackV3Token;
+    private OpenStackV3User openStackV3User;
     private Properties properties;
     private OpenStackHttpClient openStackHttpClient;
 
     @Before
     public void setUp() throws InvalidParameterException, UnexpectedException {
+        testUtils.mockReadOrdersFromDataBase();
         PropertiesHolder propertiesHolder = PropertiesHolder.getInstance();
         this.properties = propertiesHolder.getProperties();
         this.properties.put(OpenStackNetworkPlugin.KEY_EXTERNAL_GATEWAY_INFO, DEFAULT_GATEWAY_INFO);
```

Comment:
Fill in the key fields from the "cloud.conf" configuration file in the _src/test/resources/private/clouds/default_ folder and you can remove all of this configuration from the _properties_ attribute.